content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -45,6 +45,68 @@ For a full list of supported libraries and processor architectures, see [Compati
 
 ## Installation and getting started
 
+### Configure the Datadog Agent for APM
+
+Install and configure the Datadog Agent to receive traces from your now instrumented application. By default the Datadog Agent is enabled in your `datadog.yaml` file under `apm_config` with `enabled: true` and listens for trace traffic at `localhost:8126`. For containerized environments, follow the links below to enable trace collection within the Datadog Agent.
+
+{{< tabs >}}
+
+{{% tab "Containers" %}}
+
+1. Set `apm_non_local_traffic: true` in the `apm_config` section of your main [`datadog.yaml` configuration file][1].
+
+2. See the specific setup instructions to ensure that the Agent is configured to receive traces in a containerized environment:
+
+{{< partial name="apm/apm-containers.html" >}}
+</br>
+
+3. After having instrumented your application, the tracing client sends traces to `localhost:8126` by default.  If this is not the correct host and port change it by setting the below env variables:
+
+    `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`.
```

Comment:
Should we use `DD_TRACE_AGENT_URL` here? The plan some time ago was to move towards the single `DD_TRACE_AGENT_URL` instead of these two, so `DD_TRACE_AGENT_URL` overrides them if they are present. This can be confusing for users if they follow this doc.

Reply:
I'll double check, but yeah if this is the direction we're going, I can post a separate PR to do the cross-language change.